src/WebJobs.Script/Host/ProxyFunctionProvider.cs

```diff
@@ -94,7 +94,7 @@ private ImmutableArray<FunctionMetadata> LoadFunctionMetadata()
             var functionErrors = new Dictionary<string, ICollection<string>>();
             Collection<ProxyFunctionMetadata> proxies = ReadProxyMetadata(_scriptOptions.Value.RootScriptPath, _logger, functionErrors);
 
-            ImmutableArray<FunctionMetadata> metadata;
+            ImmutableArray<FunctionMetadata> metadata = ImmutableArray<FunctionMetadata>.Empty;
```

Comment:
Unrelated to the regex change. after upgrading the project to net8, this line was throwing
 _CS0165 Use of unassigned local variable 'metadata'_